Runtime/BacktraceClient.cs

```diff
@@ -413,15 +416,28 @@ private void Awake()
         /// <summary>
         /// Update native client internal ANR timer.
         /// </summary>
-        private void Update()
+        private void LateUpdate()
```

Comment:
Why is this `LateUpdate` now? What happens in `Update`?

Reply:
To give main thread to update before an anr thread

Reply:
Ah okay that makes sense